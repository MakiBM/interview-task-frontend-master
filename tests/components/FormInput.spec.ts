import { describe, it, expect, vi } from 'vitest'
import { mount } from '@vue/test-utils'
import Component from '../../src/components/FormInput.vue'

describe('FormInput Snapshot Tests', () => {
  it('matches the snapshot', () => {
    const wrapper = mount(Component, {
      props: {
        label: 'Test Label',
        name: 'Test Placeholder',
        modelValue: 'Test Value',
      },
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
