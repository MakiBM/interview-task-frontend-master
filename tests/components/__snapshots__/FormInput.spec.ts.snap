// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FormInput Snapshot Tests > matches the snapshot 1`] = `
"<div class="form-input" for="Test Placeholder" title="Test Placeholder" data-v-28dbba01=""><input class="form-input__input" type="text" placeholder="Test Label" id="Test Placeholder" data-v-28dbba01=""><!-- Use for icon -->
</div>"
`;
